recipes/pyem/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "pyem" %}
+{% set version = "0.62" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/asarnow/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a3e97c2ca83229617b04babf8912ef11e9258b225b2c3e559d654886bdc32d16
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or py<39 or py>311]
+  entry_points:
+    - cfsc.py = pyem.cli.cfsc:_main_
+    - csparc2star.py = pyem.cli.csparc2star:_main_
+    - ctf2star.py = pyem.cli.ctf2star:_main_
+    - disparticle.py = pyem.cli.disparticle:_main_
+    - emcalc.py = pyem.cli.emcalc:_main_
+    - map.py = pyem.cli.map:_main_
+    - mask.py = pyem.cli.map:_main_
+    - mcstar.py = pyem.cli.mcstar:_main_
+    - par2star.py = pyem.cli.par2star:_main_
+    - project.py = pyem.cli.project:_main_
+    - projection_subtraction.py = pyem.cli.projection_subtraction:_main_
+    - stack.py = pyem.cli.stack:_main_
+    - star.py = pyem.cli.star:_main_
+    - star2bild.py = pyem.cli.star2bild:_main_
+    - subparticles.py = pyem.cli.subparticles:_main_
+    - varmap.py = pyem.cli.varmap:_main_
+
+requirements:
+  build:
+  host:
+    - python
```

```suggestion
    - python 3.10.*
```

If you really only want Python 3.10, but I doubt that's the case.

---

Done